microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,224 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
+
+* Wide surface: Wide tile in start menu, showing a more articulated app icon: 
+![The wide size icon](../media/windows-image-wide-tile.png) 
+
+Notice how the content of the icon changed. This is possible only through the use of higher levels of icon support. We recommend developers choose higher levels to give the best experience for your Windows users.  
+
+<!-- ====================================================================== -->
+### Icon descriptions
+
+Below you’ll find a description of each app icon and where they show up in Windows.
+
+The normal app icon that's shown in the start menu, task bar, or task manager.
+
+![App Icon](../media/windows-image-app-icon.png) 
+
+* 44x44
+* 55x55 (1.25x scale)
+* 66x66 (1.5x scale)
+* 88x88 (2x scale)
+* 176x176 (4x scale)
+
+The small tile is shown in the start menu when the user set your app’s tile to small size.
+
+![Small tile](../media/windows-image-small-tile.png) 
+
+* 71x71
+* 89x89 (1.25x scale)
+* 107x107 (1.5x scale)
+* 142x142 (2x scale)
+* 284x284 (4x scale)
+
+The medium tile icon is shown in the start menu when the user sets your app’s tile to medium size.
+
+![Medium tile](../media/windows-image-medium-tile.png)
+
+* 150x150
+* 188x188 (1.25x scale)
+* 225x225 (1.5x scale)
+* 300x300 (2x scale)
+* 600x600 (4x scale)
+
+The wide tile icon is shown in the start menu when the user sets your app’s tile to wide size.
+
+![Wide tile](../media/windows-image-wide-tile.png)
+
+* 310x150
+* 388x188 (1.25x scale)
+* 465x225 (1.5x scale)
+* 620x300 (2x scale)
+* 1240x600 (4x scale)
+
+The large tile icon is in the start menu when the user sets your app’s tile to large size.
+![Large tile](../media/windows-image-large-tile.png)
+
+* 310x310
+* 388x388 (1.25x scale)  
+* 465x465 (1.5x scale)
+* 620x620 (2x scale)
+* 1240x1240 (4x scale)
+
+The store logo icon is shown in app installer, Windows Partner Center, the “Report an app” option in the Store, and the “Write a review” option in the Store.
+
+![Store Logo](../media/windows-image-store-logo.png)
+
+* 50x50
+* 63x63 (1.25x scale)
+* 75x75 (1.5x scale)
+* 100x100 (2x scale)
+* 200x200 (4x scale)
+
+The splash screen asset is shown as the splash screen for your app. Currently supported only in classic package. In the future, we may add support for the modern hosted app package as well.
+
+![Splash Screen](../media/windows-image-splash.png)
+
+* 620x300
+* 775x375 (1.25x scale)
+* 930x450 (1.5x scale)
+* 1240x600 (2x scale)
+* 2480x1200 (4x scale)
+
+### Target sized images
+
+In addition to the standard scale factor sizes described above, we also recommend creating “target-size” assets. We call these assets target-size because they target specific sizes, such as 16 pixels, rather than specific scale factors, such as 400. Target-size assets are for Windows surfaces that don’t use the scaling plateau system.
+
+![Shown in start jump list, shortcuts, control panel:](../media/windows-image-target-size.png)
```

Comment:
This image is confusing right now, because there is no text before it that explains what users are supposed to understand by looking at it.

Maybe we need to add a sentence before the image that says something like "For example, the **Apps & features** Windows setting uses app icons with specific sizes".

And I would recommend creating a new screenshot that shows the entire settings window, rather than just this cropped version.

Reply:
@captainbrosset For windows-image-target-size.pn?  This was taken from the blog?  Should we change it?

Reply:
Yes, full window images are much better for our readers than cropped versions that are hard to get appropriate context information from.
We don't need to absolutely use the images from the blog. We can make our own if needed.

Reply:
I met with Diego and we figured a good image replacement and have upload it.